eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -196,7 +209,24 @@ function Upload-Blobs
     Write-Host "DocVersion $($DocVersion)"
     Write-Host "DocDir $($DocDir)"
     Write-Host "Final Dest $($DocDest)/$($PkgName)/$($DocVersion)"
+    Write-Host "Release Tag $($ReleaseTag)"
 
+    # Use the step to replace master link to release tag link 
+    if ($ReleaseTag) {
+        foreach ($htmlFile in (Get-ChildItem $DocDir -include *.html -r)) 
+        {
+            $fileContent = Get-Content -Path $htmlFile
```

Comment:
If you pass in the "-Raw" option you can do a global replace and not have to worry about doing this a line at a time.

Reply:
Will try this to see if it works well for our case.

Reply:
I am trying to achieve several things here:
1. Replace only we identify the match.
2. Match is faster than replace, so I want to do replace when neccessary.
3. The -replace does not work as expected when we have two matches in one line. 
```
<a href="github.com/Azure/azure-sdk-for-java/tree/master/...">github.com/Azure/azure-sdk-for-java/tree/master/...</a>
```
When I checked with html, it only replace one of the link, which is not what we expect. So we have to use the while loop to achieve it.